perception/image_projection_based_fusion/config/pointpainting.param.yaml

```
@@ -0,0 +1,10 @@
+/**:
+  ros__parameters:
+    class_names: ["CAR", "PEDESTRIAN", "BICYCLE"]
+    rename_car_to_truck_and_bus: true
+    point_feature_size: 6  # x, y, z and car, pedestrian, bicycle
+    max_voxel_size: 40000
+    point_cloud_range: [-76.8, -76.8, -3.0, 76.8, 76.8, 5.0]
+    voxel_size: [0.32, 0.32, 8.0]
+    downsample_factor: 2
```

```suggestion
    downsample_factor: 1
```

Cloud you confirm that the node runs successfully by this change?

---

yes, it runs on my machine.

---

fixed at [f267845](https://github.com/autowarefoundation/autoware.universe/pull/991/commits/f2678457b824ccba4514d0b548a3dce5baf7c717)